src/components/EnterpriseApp/data/hooks/useUpdateActiveEnterpriseForUser.js

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect, useCallback } from 'react';
+import { logError } from '@edx/frontend-platform/logging';
+
+import LmsApiService from '../../../../data/services/LmsApiService';
+
+const useUpdateActiveEnterpriseForUser = ({
+  enterpriseId,
+  user,
+}) => {
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
[curious, non-blocking] Might consider what scope is involved in refactoring this hook a bit to utilize `@tanstack/react-query`? What benefits might we get from React Query in relation to the scope? If the benefits are not significant in this particular use case, OK to not utilize React Query here. 

E.g., removes boilerplate around needing to manage our own loading states, etc.